src/Exception/HttpException.php

```diff
@@ -42,7 +42,7 @@ public function __construct(
      *
      * @return ResponseInterface
```

Comment:
in our projects we started the rule that when `@return` just repeats the return type declaration and has no valuable information, we remove the `@return` line. 

(in that vein "Returns the response" is redundant to the method name. either there is more details to explain or there should be no comment...)

Reply:
I agree. 👍 

I will remove these redundant comments.